openage/testing/main.py

```diff
@@ -42,29 +42,32 @@ def init_subparser(cli):
     cli.add_argument("--demo", "-d", nargs=argparse.REMAINDER,
                      help=("run the given demo; the remaining arguments "
                            "are passed to the demo."))
+    cli.add_argument("--benchmark", "-b", nargs=argparse.REMAINDER,
+                     help=("run the given benchmark"))
     cli.add_argument("test", nargs='*', help="run this test")
 
 
 def process_args(args, error):
     """ Processes the given args, detecting errors. """
-    if not args.run_all_tests and not args.demo and not args.test:
+    if not args.run_all_tests and not args.demo and not args.test and not args.benchmark:
         args.list = True
 
     if args.have_assets and not args.run_all_tests:
         error("you have to run all tests, "
               "otherwise I don't care if you have assets")
 
-    if args.run_all_tests and (args.test or args.demo):
-        error("can't run individual test or demo when running all tests")
+    if args.run_all_tests and (args.test or args.demo or args.benchmark):
+        error("can't run individual test or demo or benchmark when running "
+              "all tests")
 
-    if args.test and args.demo:
-        error("can't run a demo _and_ tests")
+    if args.test and (args.demo or args.benchmark) or args.demo and args.benchmark:
```

Comment:
Right. I'd add more `()` for clarification though. e.g. `(args.test and (args.demo or args.benchmark)) or (args.demo and args.benchmark)`